src/snowflake/cli/_plugins/nativeapp/codegen/setup/native_app_setup_processor.py

```diff
@@ -94,7 +94,7 @@ def process(
 
         self._create_or_update_sandbox()
 
-        cc.phase("Processing Python setup files")
+        cc.step("Processing Python setup files")
```

Comment:
revert this, `phase` was intentional.

Reply:
does this actually work? 
cc.phase is supposed to be used with with:
```
with cc.phase:
```
but if I change it to that, it says nested phases are not allowed.

Reply:
right, there's an entire phase for the processors around it. You're right, the fix is correct, my bad. It should be step here, at least until we can update the console API to work better for this use case.